Sources/PowerSync/QueriesProtocol.swift

```diff
@@ -20,67 +24,37 @@ public struct WatchOptions<RowType> {
 public protocol Queries {
     /// Execute a write query (INSERT, UPDATE, DELETE)
     /// Using `RETURNING *` will result in an error.
-    func execute(sql: String, parameters: [Any]?) async throws -> Int64
-
-    /// Execute a read-only (SELECT) query and return a single result.
-    /// If there is no result, throws an IllegalArgumentException.
-    /// See `getOptional` for queries where the result might be empty.
-    func get<RowType>(
```

Comment:
Previously we had multiple definitions of the `get` method. A definition was included for versions where the `mapper` could `throw` or not. Throwing functions are a superset of non-throwing functions in Swift. A single declaration where `mapper` can throw allows passing a mapper which does not throw. I've removed these duplicates.